csrc/mlp_cuda.cu

```diff
@@ -1274,28 +1271,29 @@ int mlp_fp(
 
     // try with cublaslt first for supported case with valid handle
     int cublaslt_status = 1;
-    if(lthandle_status == CUBLAS_STATUS_SUCCESS && activation < 2){
-      cublaslt_status = mlp_gemm_lt(
-        ltHandle,
-        CUBLAS_OP_T,
-        CUBLAS_OP_N,
-        ofeat,
-        batch_size,
-        ifeat,
-        &one,
-        weight,
-        ifeat,
-        input,
-        ifeat,
-        &zero,
-        output,
-        ofeat,
-        lt_workspace,
-        1 << 22,
-        stream,
-        use_bias == 1,
-        activation == 1,
-        bias);
+    if(activation < 1){
```

Comment:
Curious was this intentional to change it to `activation < 1` instead of `activation < 2` cause now cublaslt wont get hit for relu, and this seems to kill performance for me quite a lot

Reply:
sorry to get back to you so late @ngoyal2707.
This is intentional due to some perf issue. We're working on it and we'll re-enable relu fusion once the issue get resolved.